cluster-autoscaler/cloudprovider/oci/nodepools/oci_manager.go

```diff
@@ -634,6 +647,26 @@ func addTaintToSpec(node *apiv1.Node, taintKey string, effect apiv1.TaintEffect)
 	return true
 }
 
+func getEphemeralResourceRequestsInBytes(tags map[string]string) (int64, error) {
+	for key, value := range tags {
+		if key == npconsts.EphemeralStorageSize {
+			klog.V(0).Infof("ephemeral-storage size set with value : %v", value)
+			value = strings.ReplaceAll(value, " ", "")
+
+			defer func() {
+				if err := recover(); err != nil {
+					klog.Errorf("Exception: %v", err)
+				}
+			}()
+
+			resourceSize := resource.MustParse(value)
```

Comment:
https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#ParseQuantity
We should use ParseQuantity instead of MustParse.

Reply:
Thanks, made the changes.